Yoast/yoast-components
# Fix snippet preview overflow with long text.

## Summary

This PR can be summarized in the following changelog entry:

* Improve the Snippet Preview rendering with long text.

## Relevant technical choices:

- unless I'm missing something, the description should have maximum 4 lines of text
- adjusts the max-height to a proper value, based on the line height
- also improves some indentation, as we shouldn't have lines with a mix of tab characters and spaces

## Test instructions

- to test on the standalone version, I've merged develop so the fix from #539 is in
- run `yarn start` and go on localhost:3333
- first, observe the behavior in the title field
- depending on the text added, you may see the overflowing word appear for some milliseconds and then disappear (suggestion: try repeatedly with very short words)
- then compare with the description field behavior
- the max number of lines is 4: when adding overflowing words, they disappear almost instantly (behavior should be the same of the title field)
- optionally, yarn link to the plugin and test in the meta box

Fixes https://github.com/Yoast/wordpress-seo/issues/9735
